build.gradle

```diff
@@ -58,7 +58,7 @@ repositories {
 publishing {
     publications {
         mavenJava(MavenPublication) {
-            groupId = "com.disneystreaming.oss"
+            groupId = "com.disneystreaming.smithy"
```

Comment:
alternatively you could use `mill lsp.publishLocal`, which would save us this change. Not bothered though.